lib/auth/trustedcluster.go

```diff
@@ -305,11 +305,9 @@ func (a *Server) addCertAuthorities(trustedCluster types.TrustedCluster, remoteC
 		remoteCertAuthority.SetName(trustedCluster.GetName())
 
 		// wipe out roles sent from the remote cluster and set roles from the trusted cluster
+		// TODO(zmb3) double check this
```

Comment:
@rosstimothy this look okay to you?

I'm wondering if `SetRoles(nil)` is even required.

Reply:
Why not remove the `roles` field from `CertAuthoritySpecV2` altogether? Until the field is actually gone, I'd still rather have an explicit `SetRoles(nil)` on ingest.

Reply:
Because I was afraid that might break something 😅

Happy to pursue that as well if you think it's okay.

Reply:
I'd say they're one and the same, if we remove one we should remove the other too.

Reply:
(and we should remove both)